doc/develop/west/zephyr-cmds.rst

```diff
@@ -281,3 +281,129 @@ Additional tips:
   separate ``west global`` command since ``global`` already searches for the
   ``GTAGS`` file starting from your current working directory. This is why you
   need to run ``global`` from inside the workspace.
+
+.. _west-patch:
+
+Working with patches: ``west patch``
+************************************
+
+The ``patch`` command allows users to apply patches to Zephyr or Zephyr modules
+in a controlled manner that makes automation and tracking easier for external applications that
+use the :ref:`T2 star topology <west-t2>`.
+
+There are several sub-commands available:
+
+* ``apply``: apply a patch to the Zephyr module or a module in the workspace.
+* ``clean``: remove all patches that have been applied to the Zephyr module or a module in the
+  workspace.
+* ``list``: list all patches that have been applied to the Zephyr module or a module in the
+  workspace.
+* ``gh-fetch``: fetch patches from GitHub.
```

Comment:
Ooo... I just noticed that this could probably be worded better.

```suggestion
The ``patch`` command allows users to apply patches to Zephyr or Zephyr modules
in a controlled manner that makes automation and tracking easier for external applications that
use the :ref:`T2 star topology <west-t2>`.

There are several sub-commands available to manage patches for Zephyr or other modules in the workspace.

* ``apply``: apply patches listed in ``patches.yml``
* ``clean``: remove all patches that have been applied, and reset to the manifest checkout state
* ``list``: list all patches in ``patches.yml``
* ``gh-fetch``: fetch patches from a GitHub pull request
```